src/main/java/uk/gov/pay/products/filters/LoggingMDCRequestFilter.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.pay.products.filters;
+
+import org.slf4j.MDC;
+
+import javax.ws.rs.container.ContainerRequestContext;
+import javax.ws.rs.container.ContainerRequestFilter;
+import java.io.IOException;
+import java.util.Optional;
+
+import static uk.gov.pay.logging.LoggingKeys.GATEWAY_ACCOUNT_ID;
+
+public class LoggingMDCRequestFilter implements ContainerRequestFilter {
+    
+    public static final String PAYMENT_EXTERNAL_ID = "payment_external_id";
```

Comment:
this can use the key of the same name defined in pay-java-commons

Reply:
Done